webapp/src/client/client.js

```diff
@@ -271,29 +258,61 @@ export default class Client {
             automaticLeaderAssignment: 'SameEnterprise',
         };
 
-        const response = await request.
-            post(url).
-            set('Authorization', 'Bearer ' + appAccessToken).
-            set('Accept', 'application/json').
-            send(data);
+        const response = await this.doPost(url, data, {Authorization: 'Bearer ' + appAccessToken, Accept: 'application/json'});
 
         return {
-            meetingId: response.body.onlineMeetingId,
-            meetingUrl: response.body.joinUrl,
+            meetingId: response.onlineMeetingId,
+            meetingUrl: response.joinUrl,
         };
     };
 
-    sendPost = async (url, body, headers = {}) => {
-        headers['X-Requested-With'] = 'XMLHttpRequest';
+    doGet = async (url, headers = {}, credentials) => {
+        const options = {
+            method: 'get',
+            headers,
+        };
+
+        const newOptions = Client4.getOptions(options);
+
+        if (credentials) {
+            newOptions.credentials = credentials;
+        }
+
+        const response = await fetch(url, newOptions);
+
+        if (response.ok) {
+            return response.json();
+        }
+
+        const text = await response.text();
+
+        throw new ClientError(Client4.url, {
+            message: text || '',
+            status_code: response.status,
+            url,
+        });
+    }
 
-        const response = await request.
-            post(url).
-            send(body).
-            set(headers).
-            type('application/json').
-            accept('application/json');
+    doPost = async (url, body, headers = {}) => {
+        const options = {
+            method: 'post',
+            body: JSON.stringify(body),
+            headers,
```

Comment:
One of the API calls to Skype's server fails when the `Content-Type` header is not set to `application/json`. The original code had these headers set in the `sendPost` function, which is now this `doPost` function, so adding them here is using the same logic as before, which I have asserted that it solves the issue.

0/5 on removing any reference to these headers from the callers as well.

```suggestion
            headers: {
                ...headers,
                'Content-Type': 'application/json',
                 Accept: 'application/json',
            },
```